src/styles/utils/_variable.scss

```diff
@@ -0,0 +1,19 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap");
+
+$ff: "Roboto", sans-serif;
+$fw-400: 400;
+$fw-500: 500;
+$fw-700: 700;
+$fs-100: 10px;
+$fs-200: 12px;
+$fs-400: 14px;
+$fs-600: 16px;
```

Comment:
better to come up with primary/secondary and sth like that, to me it is not really clear now what the variable actually means